src/OpenTelemetry.Contrib.Instrumentation.AWS/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog - OpenTelemetry.Contrib.Instrumentation.AWS
 
+## Unreleased
+
+* Update the `ActivitySource` name to the assembly name:
```

Comment:
might be a breaking change, since this package is already stable.

Reply:
> might be a breaking change, since this package is already stable.

Does it make sense to remove the unreleased section?

Reply:
Can I get an update on this?

Reply:
Since this is a breaking change, you should wait for the `2.0.x` release to make this change. This change shouldn't go into `1.x.x` versions.

Reply:
There was a decision to remove ".Contrib" from all packages. Only AWS packages do not follow this change as mentioned in [this comment](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/534#discussion_r934135417)

Reply:
This package was only ever released as a beta, so it is not "already stable", or am I missing something @cijothomas? 1.1.0-beta1 is also what I see at NuGet: https://www.nuget.org/packages/OpenTelemetry.Contrib.Instrumentation.AWSLambda/